tests/README.md

```diff
@@ -41,57 +41,47 @@ This directory contains E2E tests which are run against BotKube installed on Kub
 5. Navigate to the **OAuth & Permissions** section
 6. Copy the **Bot User OAuth Token** and save it for later.
 
-  You can already export it as environment variable for [running the tests locally](#run-tests-locally):
+   You can already export it as environment variable for [BotKube installation](#install-botkube) or [running the tests locally](#run-tests-locally):
+   
+   ```bash
+   export SLACK_TESTER_APP_TOKEN="{BotKube tester app token}
+   ```
 
-  ```bash
-  export SLACK_TESTER_APP_TOKEN="{BotKube tester app token}
-  ```
+## Install BotKube
 
-## Run tests locally
+1. Export required environment variables:
 
-To run the tests manually against the latest development version, follow these steps:
+    ```bash
+    export SLACK_BOT_TOKEN="{token for your configured BotKube app}" # WARNING: It is token for BotKube Slack bot, not the Tester!
+    export IMAGE_REGISTRY="ghcr.io"
+    export IMAGE_REPOSITORY="infracloudio/botkube"
+    export IMAGE_TAG="v9.99.9-dev"
+
+    #
+    # The following environmental variables are required only when running integration tests via Helm:
+    #
+    export SLACK_TESTER_APP_TOKEN="{BotKube tester app token}" # WARNING: This is a token for Tester, not the BotKube Slack bot!
+    export TEST_IMAGE_REGISTRY="docker.io" # TODO: Replace with image built on `develop`
+    export TEST_IMAGE_REPOSITORY="pkosiec/botkube-test" # TODO: Replace with image built on `develop`
+    export TEST_IMAGE_TAG="latest" # TODO: Replace with image built on `develop`
```

Comment:
I will replace this in a follow-up PR after merging this PR and building test image.